roborock/version_1_apis/roborock_local_client_v1.py

```diff
@@ -53,15 +54,19 @@ async def send_message(self, roborock_message: RoborockMessage):
             response_protocol = request_id + 1
```

Comment:
Is this trying to set the response protocol to the next thing, e.g. HELLO_REQUEST to HELLO_RESPONSE?  I think it ends up munging things up and i can't tell what this is trying to do.

Reply:
Yes
I believe that it might be a mapping for dps request and response. But since I didn't have that I noticed that the response protocol was always a number higher

Reply:
OK great, maybe i can figure out another way to use the new protocol types. I'll think about this more and maybe:
(1) Add some tests to exercise these cases explicitly
(2) address first as a separate change
(3) come back to this